src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -19,12 +19,10 @@
     <DelaySign>false</DelaySign>
     <AssemblyOriginatorKeyFile>35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <VersionPrefix>1.20.0</VersionPrefix>
+    <VersionPrefix>1.20.1</VersionPrefix>
```

Comment:
i think 1.20.1 is correct but the title is indicating this should be 1.21.1

Reply:
Fixed that. https://www.nuget.org/packages/Microsoft.Graph.Core 1.20.0 is the last release for this major version.